system-addon/lib/ActivityStream.jsm

```diff
@@ -81,6 +82,21 @@ const PREFS_CONFIG = new Map([
       "provider_name": "Pocket",
       "provider_icon": "pocket"
     }`
+  }],
+  ["migrationExpired", {
+    title: "Boolean flag that will always be true once the migration period has passed.",
```

Comment:
Not sure if this is better named as "migrationDismissed" or "migrationDone" or as it is right now could be fine. The description is not as accurate in that there's multiple ways that it becomes `true`.